Framework/Algorithms/src/ReflectometryReductionOneAuto.cpp

```diff
@@ -462,21 +479,35 @@ void ReflectometryReductionOneAuto::exec() {
     if (theta_in.is_initialized()) {
       refRedOne->setProperty("ThetaIn", theta_in.get());
     }
-
+    double scaleFactor = getProperty("ScaleFactor");
+    if (scaleFactor != 1.0) {
+      refRedOne->setProperty("ScaleFactor", scaleFactor);
+    }
+    auto momentumTransferMinimum = isSet<double>("MomentumTransferMinimum");
+    auto momentumTransferStep = isSet<double>("MomentumTransferStep");
+    auto momentumTransferMaximum = isSet<double>("MomentumTransferMaximum");
+    if (momentumTransferMinimum.is_initialized() &&
+        momentumTransferMaximum.is_initialized() &&
+        momentumTransferStep.is_initialized()) {
+      refRedOne->setProperty("MomentumTransferMinimum",
+                             momentumTransferMinimum.get());
+      refRedOne->setProperty("MomentumTransferStep",
+                             momentumTransferStep.get());
+      refRedOne->setProperty("MomentumTransferMaximum",
+                             momentumTransferMaximum.get());
+    }
```

Comment:
If `MomentumTransferMinimum`, `MomentumTransferStep` and `MomentumTransferMaximum` are not set, we should try to calculate them. To do so, we would need to apply the formula:

```
Q = 4 * pi / lambda * sin (theta)
```
to get the minimum and maximum values, and run `CalculateResolution` to get the step. But this is only possible if `ThetaIn` has been supplied by the user. If `ThetaIn` has not been set, we will have to wait until `SpecularReflectionCalculateTheta` is run, which happens in `ReflectometryReductionOne`.